user/common-tasks/copy-paste.md

```diff
@@ -64,6 +73,10 @@ For example, if you are certain that you never wish to paste *into* your "vault"
 @anyvm  @anyvm  ask
 ~~~
 
+Note that for historical reasons, the Qubes clipboard paste shortcut treats `ask` in this policy file as `allow` (as the comment at the top of the file states); there is currently no way to get it to show a prompt, as `ask` in any other policy file would do.
+There is [an enhancement proposal](https://github.com/QubesOS/qubes-issues/issues/6347) to add an option in `/etc/qubes/guid.conf` to make `ask` show a prompt.
+This could be used as an alternative or additional mitigation for the focus stealing risk described above.
```

Comment:
The documentation should describe the current state, not something that may eventually be implemented in the future but perhaps differently.  IOW, this paragraph does not belong in this file until the actual feature gets implemented (and then - should be marked at which version it is available),

Reply:
Fair enough.